.github/workflows/gke-vpa-recommendations-ci.yml

```diff
@@ -19,7 +19,8 @@ jobs:
         run: |
           cd gke-vpa-recommendations/metrics-exporter
           docker build --tag metric-exporter .
-      - name: build hpa-metrics-exporter container
+      - name: Validate terraform
         run: |
-          cd gke-vpa-recommendations/scripts
```

Comment:
Is that image not used for the tutorial anymore? If so perhaps we should delete it (and relevant obsolete files) from here?

https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/58208b054815740b4fc8b53f11053e0b162d7a94/gke-vpa-recommendations/scripts/

Reply:
no its no longer used in the tutorial

Reply:
also removed the files supporting the hpa metric exporter